packages/mdc-ripple/README.md

```diff
@@ -30,6 +30,9 @@ to provide components (or any element at all) with a
 material "ink ripple" interaction effect. It is designed to be efficient, un-invasive, and usable
 without adding any extra DOM to your elements.
 
+and includes a gracefully degraded version that can be used
```

Comment:
This paragraph looks a bit broken...

Maybe we should say:

MDC Ripple also works without javascript, where it gracefully degrades to a simpler CSS-Only implementation.